spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -437,20 +480,33 @@ private RabbitTemplate buildRabbitTemplate(RabbitProducerProperties properties)
 	 * @param baseQueueName   The base name for the queue (including the binder prefix, if any).
 	 * @param routingKey  The routing key for the queue.
 	 * @param autoBindDlq true if the DLQ should be bound.
+	 * @param properties the properties.
 	 */
-	private void autoBindDLQ(final String baseQueueName, String routingKey, String prefix, boolean autoBindDlq) {
+	private void autoBindDLQ(final String baseQueueName, String routingKey, RabbitCommonProperties properties) {
```

Comment:
Time to move to IDEA?
![wrongjavadoc](https://cloud.githubusercontent.com/assets/1071590/22026684/8736fd30-dc9f-11e6-86b6-1dc18bf0be96.png)

😄